Documentation/op-guide/supported-platform.md

```diff
@@ -32,7 +32,7 @@ For etcd to officially support a new platform as stable, a few requirements are
 
 etcd has known issues on 32-bit systems due to a bug in the Go runtime. See the [Go issue][go-issue] and [atomic package][go-atomic] for more information.
 
-To avoid inadvertently running a possibly unstable etcd server, `etcd` on unstable or unsupported architectures will print a warning message and immediately exit if the environment variable `ETCD_UNSUPPORTED_ARCH` is not set to the target architecture.
+To avoid inadvertently running a possibly unstable etcd server, `etcd` on unstable or unsupported architectures will print a warning message and immediately exit if flag `--unsupported-arch` or environment variable `ETCD_UNSUPPORTED_ARCH` is not set to the target architecture.
```

Comment:
there is no flag for this feature since we do not really want to promote the usage of running etcd on unsupported platform, and do not want to make it a feature flag. env is much more easier to deprecated for example.

Reply:
Then should we just have special handling for `ETCD_UNSUPPORTED_ARCH`? Otherwise, we will keep warning

> pkg/flags: unrecognized environment variable ETCD_UNSUPPORTED_ARCH=arm64

Reply:
yes possiblely.